content/en/docs/concepts/scheduling-eviction/taint-and-toleration.md

```diff
@@ -80,7 +80,7 @@ There are two special cases:
 An empty `key` with operator `Exists` matches all keys, values and effects which means this
 will tolerate everything.
 
-An empty `effect` matches all effects with key `key1`.
+An empty `effect` matches all effects with key `key1`(the `operator` is `Equal` and the `values` are equal).
```

Comment:
I don't think this is true. Leaving `effect` unset does not force the `operator` to be Equal.

Can you provide evidence that I've got it wrong? If not, I don't think we should merge the change.